lib/mixlib/shellout.rb

```diff
@@ -322,7 +363,8 @@ def parse_options(opts)
           self.log_tag = setting
         when 'environment', 'env'
           self.environment = setting || {}
-
+        when 'with_logon'
```

Comment:
Sure, I didn't want to rename variables. `login` sounds great, since "shell" might imply using the actual user's home shell (the one specified in the `passwd` file) when we're doing no such thing.

Should I rename it everywhere?